catroid/src/main/res/values/strings.xml

```diff
@@ -1801,6 +1801,8 @@ needs read and write access to it. You can always change permissions through you
     <string name="formula_editor_function_list_item_parameter">(1,*list name*)</string>
     <string name="formula_editor_function_contains">contains</string>
     <string name="formula_editor_function_contains_parameter">(*list name*,1)</string>
+    <string name="formula_editor_function_index_of_item">index of item</string>
+    <string name="formula_editor_function_index_of_item_parameter">(1,*list name*)</string>
```

Comment:
pls check again if it is possible with apostroph

Reply:
Renamed it, so that it is equal as in the ticket description